src/documentation/chunk.rs

```diff
@@ -213,6 +213,23 @@ impl CheckableChunk {
             .collect::<IndexMap<_, _>>()
     }
 
+    pub fn find_spans_inclusive(&self, range: Range) -> Vec<Span> {
```

Comment:
Docs :question:

Reply:
And why inclusive?

Reply:
And if inclusive, we should probably use `RangeInclusive` (roughly, iirc)

Reply:
That was more a working title.. I'm really bad at naming things though!

Inclusive should indicate that the all spans at least partially contain range, so all spans together contain the range if that makes any sense. I'll document this and tried to come up with a better name